core/src/main/java/io/grpc/LoadBalancer2.java

```diff
@@ -181,12 +206,59 @@ private PickResult(Subchannel subchannel, Status status) {
     }
 
     /**
-     * A decision to proceed the RPC on a Subchannel.  The state of the Subchannel is supposed to be
-     * {@link ConnectivityState#READY}.  However, since such decisions are racy, a non-READY
-     * Subchannel will not fail the RPC, but will only leave it buffered.
+     * A decision to proceed the RPC on a Subchannel.
      *
      * <p>Only Subchannels returned by {@link Helper#createSubchannel} will work.  DO NOT try to
      * use your own implementations of Subchannels, as they won't work.
+     *
+     * <p>When the RPC tries to use the return Subchannel, which is briefly after this method
+     * returns, the state of the Subchannel will decide where the RPC would go:
+     *
+     * <ul>
+     *   <li>READY: the RPC will proceed on this Subchannel.</li>
+     *   <li>IDLE: the RPC will be buffered.  Subchannel will attempt to create connection.</li>
```

Comment:
Make a note that the LB must change the picker in the future for re-selection after the buffering here?